butterknife/build.gradle

```diff
@@ -33,9 +33,23 @@ dependencies {
   testCompile deps.junit
   testCompile deps.truth
   testCompile deps.compiletesting
-  testCompile files(org.gradle.internal.jvm.Jvm.current().getRuntimeJar())
+  testCompile files(getRuntimeJar())
   testCompile files(org.gradle.internal.jvm.Jvm.current().getToolsJar())
   testCompile project(':butterknife-compiler')
 }
 
+def getRuntimeJar() {
```

Comment:
@JakeWharton `org.gradle.internal.jvm.Jvm.current().getRuntimeJar()` was removed. See here: https://groups.google.com/forum/#!topic/sonarqube/DnE1npMpaiA and https://github.com/bfarka/sonar-gradle/blob/295c365f459cd19617b4fe8ce62bdb58b999bd74/src/main/java/org/sonarqube/gradle/SonarQubePlugin.java#L370.